rtos/rtx5/TARGET_CORTEX_M/rtx_lib.c

```diff
@@ -595,7 +595,6 @@ void *__user_perthread_libspace (void) {
 typedef void *mutex;
 
 // Initialize mutex
-__USED
```

Comment:
@0xc0170 I think you fixed issues with the tests with ops (malloc?) not being thread safe due to these symbols being stripped out?